ansible/roles/etcd/tasks/etcd2-configure.yml

```diff
@@ -11,10 +11,3 @@
 - name: CoreOS | Write etcd config file
   template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
   notify: restart etcd2
```

Comment:
From [2] citing: "Registered variables are valid on the host the remainder of the playbook run, which is the same as the lifetime of “facts” in Ansible."

So once defined they live forever. However, ``etcd2_started`` is defined in ``start.yml`` after it is used.

[2] http://docs.ansible.com/ansible/playbooks_variables.html#registered-variables